.github/workflows/tests.yml

```diff
@@ -0,0 +1,448 @@
+name: Open Runtimes Tests
+
+on: [push, pull_request, release]
+
+jobs:
+  OPEN_RUNTIMES:
+    strategy:
+        fail-fast: false
+        matrix:
+          runtimes: [node-20.0.js, node-20.0.mjs, node-19.0.js, node-19.0.mjs, node-18.0.js, node-18.0.mjs, node-16.0.js, node-16.0.mjs, node-14.5.js, node-14.5.mjs, deno-1.35, deno-1.24, deno-1.21, python-3.11, python-3.10, python-3.9, python-3.8, dart-3.0, dart-2.19, dart-2.18, dart-2.17, dart-2.16, dart-2.15, ruby-3.2, ruby-3.1, ruby-3.0, php-8.2, php-8.1, php-8.0, swift-5.8, swift-5.5, kotlin-1.6, kotlin-1.8, java-18.0, java-17.0, java-11.0, java-8.0, cpp-17, cpp-20, dotnet-7.0, dotnet-6.0]
+          include:
```

Comment:
Do we need the runtimes or can we just use include? I read this: https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#example-adding-configurations.

Otherwise, it would be good to try and figure out a combination that would cut down on the includes array a bit. Like maybe runtimes would be just the language and then include adds/expands to include different versions? Maybe not..I'm not really sure, but it would be nice if we can reduce some of the repetition in here.

Reply:
@stnguyen90 
> Do we need the runtimes or can we just use include? I read this: https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#example-adding-configurations.

Changed from using runtimes array to just include. 

> Otherwise, it would be good to try and figure out a combination that would cut down on the includes array a bit. Like maybe runtimes would be just the language and then include adds/expands to include different versions? Maybe not..I'm not really sure, but it would be nice if we can reduce some of the repetition in here.

How about having different test files like node.yml, deno.yml for each runtime as each version of node has to run twice (for tests.js and tests.mjs), few runtimes exclude ceratain architecture, we can't have an array of language and another for versions conditionally coz arrays inside -include don't work.(below mentioned code) [My Attempt](https://github.com/faisalill/open-runtimes/actions/runs/5796604844/job/15710511738)
```yml
...
matrix: 
    languages: [node,deno,python...]
    include: 
        - languages: node
          versions: [20.0, 19.0, 18.0...]
```

So how about having different .yml files for each language??